lib/iris/tests/unit/fileformats/grib/grib1_load_rules/test_grib1_convert.py

```diff
@@ -14,70 +14,37 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with Iris.  If not, see <http://www.gnu.org/licenses/>.
-"""Unit tests for :func:`iris.fileformats.grib.load_rules.convert`."""
+"""
+Unit tests for :func:`iris.fileformats.grib._grib1_load_rules.grib1_convert`.
+
+"""
 
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
 
-# Import iris tests first so that some things can be initialised before
+# Import iris.tests first so that some things can be initialised before
 # importing anything else
 import iris.tests as tests
 
 import cf_units
 import gribapi
+import mock
 
-import iris
+import iris.aux_factory
+import iris.coords
+from iris.exceptions import TranslationError
+from iris.fileformats.grib import GribWrapper
 from iris.fileformats.rules import Reference
-from iris.tests import mock
-from iris.tests.test_grib_load_translations import TestGribSimple
-from iris.tests.unit.fileformats import TestField
+from iris.fileformats.grib._grib1_load_rules import grib1_convert
+from iris.tests.unit.fileformats.grib import TestField
```

Comment:
@lbdreyer I'll push a PR to your branch with fixes to address minor review actions.